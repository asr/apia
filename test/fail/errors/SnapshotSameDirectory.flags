--atp=e --output-dir=/tmp --snapshot-test --snapshot-dir=/tmp
