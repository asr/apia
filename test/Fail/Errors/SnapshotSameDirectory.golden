ret > ExitFailure 1
err > apia: the ‘--output-dir’ and ‘--snapshot-dir’ options cannot be the same
err >
